infer_structcol/inference.py

```diff
@@ -77,42 +97,60 @@ def run_mcmc(data, sample, nwalkers, nsteps, theta = None, seed=None):
         number of parallelized MCMC walkers to use
     nsteps: int
         number of steps taken by each walker
-    theta: 3 or 5-tuple of floats (optional)
-        user's best guess of the expected parameter values (phi, l0, l1) or 
-        (phi, l0, l1, l0, l1)
+    theta_guess: list of floats (optional)
+        user's best guess of the expected parameter values (phi, radius, 
+        thickness)
+    theta_range: 2 by 3 array of floats (optional)
```

Comment:
theta_guess and theta_range are now dictionaries, but the user can set the values for them. If the user doesn't pass in any values, then the default values are used.